### DISCUSSION TITLE
Calling distributed workflow

### DISCUSSION THREAD

**Discussion by ravisamavedula** [run-llama/llama_deploy#523] [Q&A]
I have 3 workflows running in 3 different containers if workflow A needs to call workflow B and workflow B calls workflow C, how should be called, take a use case of Coder as Workflow A, who generates the code and passes the code to review to Workflow B, and after review, Workflow B passes to Tester(workflow c), there is a huge transfer of data, is this use case suitable with Llama-Deploy.

**Comment by logan-markewich** [comment]
If there is a huge transfer of data, then each workflow should be reading the data from some db or s3 storage, rather than passing over the network

**Comment by logan-markewich** [comment]
otherwise, seems like a perfectly valid use case

**Comment by ravisamavedula** [comment]
@logan-markewich  Can we add a docker sample for distributed agentic workflows, like different personas, Coder, reviewer, Tester, it will make users to adopt and test quickly, basic and simple one.
